src/mutate/index.js

```diff
@@ -39,7 +39,7 @@ const decorateCreateBase = args => {
   const mutationVars = processMutationVars(args, "create");
   const mutation = gqlMutate(mutationVars, args.fields);
   const onSubmitName =
-    args.submitKey || args.formName ? `${args.formName}OnSubmit` : `onSubmit`;
+    args.submitKey || (args.formName ? `${args.formName}OnSubmit` : `onSubmit`);
```

Comment:
```suggestion
    args.submitKey || (args.formName && `${args.formName}OnSubmit`) || `onSubmit`;
```

Reply:
I think the ternary thing is just a land mine for future devs.

If I had to do this again, I'd actually rather not have any of this as a part of qewl, and just point people towards  recompose's `mapProps`.